dev-libs/libdbi-drivers/libdbi_drivers-0.9.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Drivers for the database-independent abstraction layer written in C"
+DESCRIPTION="libdbi implements a database-independent abstraction layer in C, similar to the \
+DBI/DBD layer in Perl. Writing one generic set of code, programmers can leverage the power of \
+multiple DBs and multiple simultaneous DB connections by using this framework."
+HOMEPAGE="https://libdbi-drivers.sourceforge.net/"
+COPYRIGHT="2001-2013, David Parker, Mark Tobenkin, Markus Hoenicka"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/libdbi-drivers/files/libdbi-drivers/libdbi-drivers-$portVersion/libdbi-drivers-$portVersion.tar.gz"
+CHECKSUM_SHA256="43d2eacd573a4faff296fa925dd97fbf2aedbf1ae35c6263478210c61004c854"
+SOURCE_DIR="libdbi-drivers-$portVersion"
+PATCHES="libdbi_drivers-0.9.0.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	libdbi_drivers = $portVersion
+	"
+REQUIRES="
+    haiku
+    lib:libdbi
+    lib:libsqlite3
+    "
+
+BUILD_REQUIRES="
+    haiku_devel
+    devel:libdbi
+    devel:libsqlite3
+    "
+BUILD_PREREQUIRES="
+    cmd:automake
+    cmd:autoreconf
+    cmd:gcc
+    cmd:libtoolize
+    cmd:make
+	"
+
+BUILD()
+{
+	autoreconf -if
+	runConfigure --omit-dirs "binDir" ./configure \
+		--prefix=$prefix \
```

Comment:
prefix should be set by `runConfigure` already.

Reply:
Thanks, fixed.